src/commands/utilities.ts

```diff
@@ -31,7 +31,7 @@ export async function executeTaskWithUI(
     task: vscode.Task,
     description: string,
     folderContext: FolderContext,
-    showErrors = false,
+    showErrors = true,
```

Comment:
was this intentional or just for development? Let's chat before flipping the default here

Reply:
Ooops, good catch!